bip-p2qrh.mediawiki

```diff
@@ -85,6 +85,7 @@ Quantum resistant signature algorithms ordered by shortest proposed signature si
 * [https://sphincs.org SPHINCS+] - 17KB
 * [https://eprint.iacr.org/2014/457.pdf pqNTRUsign] - 702 bytes
 * [https://falcon-sign.info FALCON] - 666 bytes
+* [https://eprint.iacr.org/2022/1155.pdf HAWK] - 652 bytes
```

Comment:
This is fine, as long as you are okay putting a lattice based signature algorithm as an outlier in this list.

Reply:
These are all lattice-based. Even quaternion isogenies make use of lattice cryptography.

Reply:
Oh, I did not realize that. Then this is perfectly fine.